### DISCUSSION TITLE
Please change the state of issues that have been merged.

### DISCUSSION THREAD

**Discussion by lyx-jay** [opentiny/tiny-vue#384] [General]
I noticed that some issues have been resolved, but their status has not been updated in time. I hope that the status can be modified in time, which will help more developers to view issues

**Comment by kagol** [comment]
> I noticed that some issues have been resolved, but their status has not been updated in time. I hope that the status can be modified in time, which will help more developers to view issues

I agree.

**Reply by mengqiuleo** [reply]
在提交 pr 时，评论对应的 issue。参考：[Linking a pull request to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

eg: `close #1293`

关键词：close、fix...

需要改一下 贡献指南

**Reply by kagol** [reply]
好主意，已提交 PR #1382

**Reply by mengqiuleo** [reply]
这个办法好像没生效，不知道为什么，
我之前提交了骨架屏的pr,但是issue没有自动关闭。
- pr：https://github.com/opentiny/tiny-vue/pull/1345
- issue：https://github.com/opentiny/tiny-vue/issues/355

**Reply by kagol** [reply]
需要在 PR 描述里面写上 #355，而不是在评论里面写
![image](https://github.com/opentiny/tiny-vue/assets/9566362/e3bfdbb7-1eda-4b06-a3dd-6cff9cef66f2)

**Reply by kagol** [reply]
可以参考链接：https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

![image](https://github.com/opentiny/tiny-vue/assets/9566362/2fae0f91-e757-476c-b4aa-c2b9ae807379)

**Reply by mengqiuleo** [reply]
ok, 明白了，谢谢kagol老师
